modules/core/client/components/Board.js

```diff
@@ -0,0 +1,14 @@
+import { Children, cloneElement, useEffect } from 'react';
+import { selectPhoto } from '../services/photos.service';
+
+export default function Board({ name='bokeh', children, onNameChanged }) {
+
+  // update
+  useEffect(() => {
+    onNameChanged({ name: selectPhoto(name) });
+  }, [name]);
+
+  const photo = selectPhoto(name);
```

Comment:
.. and I guess missing support for:
- `tr-boards-ignore-small`
- `window.innerWidth <= 480`

... which in this case is because they're not used on the tribes header, but maybe add a note that it doesn't exactly provide the same functionality as the `tr-boards` directive.

Reply:
added the TODO for future implementation of tr-boards-ignore-small in 76d84e9